docs/resources/cluster_v2.md

```diff
@@ -461,6 +461,8 @@ The following attributes are exported:
 * `machine_config` - (Required) Machine pool node config (list)
 * `control_plane_role` - (Optional) Machine pool control plane role? (bool)
 * `etcd_role` - (Optional) Machine pool etcd role? (bool)
+* `drain_before_delete` - (Optional) Machine Pool Drain Before Delete? (bool)
+* `node_drain_timeout` - (Optional) Machine Pool Drain Timeout? (int)
```

Comment:
Can we add `node_startup_timeout_seconds`, `unhealthy_node_timeout_seconds`, `max_unhealthy`, and `unhealthy_range` as well? And also a more descriptive message for `node_drain_timeout` would be nice. Also I know it was messed up before, but if we could unify the indentation here that would be nice as well.

Reply:
I can add the missing fields, but are we sure we want unified indentation? This would be the only place in the file where that is done, and I'm not sure if it effects any generated documentation.